books/projects/translators/l3-to-acl2/examples/mips/mips.lisp

```diff
@@ -8,7 +8,8 @@
                                                ("DATA" . IS_DATA)
                                                ("gpr" . GPREGS)))))
 
-(INCLUDE-BOOK "/Users/kaufmann/projects/l3-to-acl2/translator/l3")
+(INCLUDE-BOOK
```

Comment:
It can be replaced by relative path or by `:dir :system`

Reply:
Maybe it is possible to change definition of `*tranlator-dir*` in line 1281 of `l3-to-acl2.lisp` by `projects/translators/l3-to-acl2/translator/` and add `:dir :system` in line 1293 ?

Reply:
I think that absolute path remains from time when l3-to-acl2 was not in the repository.
Now it is in the repository and `:dir :system` can be used`. Let's ask Matt.

Reply:
OK, I think you're right. I'll make a commit which does that.